src/index.html

```diff
@@ -1,14 +1,82 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+    href="https://fonts.gstatic.com" crossorigin="linthtml">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;600;700&display=swap"
+      rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+
+      <div class="header-content">
+        <div class="logo">
+          <a class="nav-link" href="#home"><img src="./images/logo.png" alt="Moyo logo"></a>
+        </div>
+
+        <nav class="nav">
+          <ul class="nav-list">
+
+            <li class="nav-list-block">
+              <a class="nav-link is-active" href="#apple">
+                Apple
+              </a>
+            </li>
+
+            <li class="nav-list-block">
+              <a class="nav-link" href="#samsung">
+                Samsung
+              </a>
+            </li>
+
+            <li class="nav-list-block">
+              <a class="nav-link" href="#smartphones">
+                Smartphones
+              </a>
+            </li>
+
+            <li class="nav-list-block">
+              <a class="nav-link" href="#laptop&computers"
```

Comment:
Please read again point 13 in checklist 
![image](https://github.com/mate-academy/layout_moyo-header/assets/54936044/e64d1752-c1cd-45ce-a3ba-97e68c010cd7)